### ISSUE TITLE
Dark mode for i18n pages and articles

### ISSUE THREAD

**Issue by xfq** [w3c/i18n-drafts#538]
We can consider adding dark mode for our website. I made an incomplete prototype so that you can see what it looks like:

https://raw.githack.com/w3c/i18n-drafts/xfq/theme-toggle/nav/about.html

**Comment by xfq** [comment]
One problem that my prototype has not solved is that I did not make images for dark mode.

**Comment by xfq** [comment]
One solution might be to put dark mode in a separate CSS file and only introduce it on pages/articles that we confirm that there is no problem, rather than introducing it to all pages/articles at once. @r12a What do you think?

**Comment by r12a** [comment]
I wonder whether a possible option would be to use CSS for images in dark mode:

`filter: invert(100%);`

**Comment by xfq** [comment]
In some cases it is fine, but it may not work for all images and needs to be checked manually.

**Comment by r12a** [comment]
I think the trick would be to add a class name to those that do work. Then you can turn on the styling when switching to dark mode.  For example, i think it would work for the bidi image at https://raw.githack.com/w3c/i18n-drafts/xfq/theme-toggle/nav/about.html

**Comment by xfq** [comment]
Yeah, similar to https://github.com/w3c/clreq/pull/612/files#diff-ca4338c57509b61601bb5299159386f2ebad02d97ba8cc5cfd11db4263a1c395R82

**Comment by xfq** [comment]
I'll work on this after working on https://github.com/w3c/i18n-drafts/pull/750
